@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -10,24 +13,33 @@ interface ParentState {
 }
 
 interface State {
+  confirmCtrlAltDel: boolean
   container: React.RefObject<HTMLDivElement>
+  // unknown type throw an error
+  RFB: any
 }
 
 interface Props {
   vmId: string
+  setCtrlAltDel: (fn: () => void) => void
 }
 
 interface ParentEffects {}
 
-interface Effects {}
+interface Effects {
+  sendCtrlAltDel: () => void
+  toggleCtrlAltDel: () => void
+}
 
 interface Computed {}
 
 // https://github.com/novnc/noVNC/blob/master/docs/API.md
 const Console = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
   {
     initialState: () => ({
+      confirmCtrlAltDel: false,
       container: React.createRef(),
+      RFB: null,
```

Comment:
Why `null` instead of `undefined`?

Reply:
`Null` instead of `undefined` because i find that weird to declare a variable and say it's undefined variable.
Since i have to declare a variable but i dont know his value at the moment, for me `null` is best thing to attribute.